packages/components/src/public/files/TemplateDownloadButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { FC, memo } from 'react';
+import { User } from '../../internal/components/base/models/User';
+import { RequiresPermission } from '../../internal/components/base/Permissions';
+import { PermissionTypes } from '@labkey/api';
+
+interface Props {
+    templateUrl?: string,
+    className?: string,
+    text?: string,
+    onClick?: () => void,
+    user?: User,
```

Comment:
is the `user` prop required?

Reply:
I suppose if it is undefined then the perm check will return false, so maybe not technically required.